### PR TITLE
Support for Plan Group API  [ch19227]

### PR DESCRIPTION
**Pull request by velinarusjakova**
* add PlanGroup endpoints

```
GET /v1/plan_groups
GET /v1/plan_groups/plan_group_uuid
GET /v1/plan_groups/plan_group_uuid/plans
POST /v1/plan_groups/plan_group_uuid
PATCH /v1/plan_groups/plan_group_uuid
DELETE /v1/plan_groups/plan_group_uuid
```

* increment version
* add changes to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,16 @@ ChartMogul.Plan.all(config, query)
 ChartMogul.Plan.destroy(config, uuid)
 ```
 
+#### [Plan Groups](https://dev.chartmogul.com/docs/plan_groups)
```
Comment (velinarusjakova) [inline README.md]: this actually doesn't exist still

File: test/chartmogul/plan-group.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    };
+    /* eslint-enable camelcase */
+
+    nock(config.API_BASE)
+      .post('/v1/plan_groups', postBody)
+      .reply(200, {
+        /* eslint-disable camelcase */
+        name: 'Bronze Plan Group',
+        uuid: 'plg_eed05d54-75b4-431b-adb2-eb6b9e543206',
+        plans: [
```
Comment (pkopac) [inline test/chartmogul/plan-group.js]: [API design] how many plans can be in the array? If there's a limit, how can it be paged? I would expect similar response to `GET`.

File: README.md (hunk 1 above)
Comment (pkopac) [reply README.md]: not a problem, you can merge on all client libraries once it's deployed ;) We'll just coordinate the release.

File: test/chartmogul/plan-group.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    });
+
+    return PlanGroup.all(config)
+    .then(res => {
+      expect(res).to.have.property('plan_groups');
+      expect(res.plan_groups).to.be.instanceof(Array);
+    });
+  });
+
+  it('should modify a plan group', () => {
```
Comment (pkopac) [inline test/chartmogul/plan-group.js]: What do I do if I want to add/remove plans from groups?

Comment (velinarusjakova) [reply test/chartmogul/plan-group.js]: I'll add the specs 💃

File: test/chartmogul/plan-group.js (hunk 2 above)
Comment (velinarusjakova) [reply test/chartmogul/plan-group.js]: Right, need to figure that one out 🍡

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "chartmogul-node",
-  "version": "1.0.11",
+  "version": "1.0.12",
```
Comment (tommycarney) [inline package.json]: I'd set this to 1.1.0 as a minor release

File: README.md [hunk 5]
```diff
@@ -133,6 +133,17 @@ ChartMogul.Plan.all(config, query)
...
+
+```js
+ChartMogul.PlanGroup.create(config, data)
+ChartMogul.PlanGroup.retrieve(config, planGroupUuid)
+ChartMogul.PlanGroup.modify(config, planGroupUuid, data)
+ChartMogul.PlanGroup.all(config, query)
+ChartMogul.PlanGroup.all(config, planGroupUuid,{ page: 1, per_page: 20 }, query)
```
Comment (hassansin) [inline README.md]:
```suggestion
ChartMogul.PlanGroup.all(config, planGroupUuid, query)
```

**Pull request merged**